kapacitor/content.md

```diff
@@ -1,45 +1,112 @@
 # Kapacitor
 
-Kapacitor is a data processing engine. It can process both stream and batch data.
+Kapacitor is an open source data processing engine written in Go. It can process both stream and batch data.
+
+[Kapacitor Official Docs](https://docs.influxdata.com/kapacitor/latest/introduction/getting_started/)
```

Comment:
From the other PR, it seems like they like using the word "Documentation" instead.